angr/analyses/cfg/cfg_accurate.py

```diff
@@ -5,6 +5,7 @@
 import claripy
 import networkx
 import pyvex
+import sys
```

Comment:
Please take this time to reorder the sequence of import. Things should go like the following:

- System packages
- Third-party packages
- Local imports